Alliance.Server/Patch/HarmonyPatch/Patch_Module.cs

```diff
@@ -0,0 +1,51 @@
+﻿using HarmonyLib;
+using System;
+using System.Reflection;
+using static Alliance.Common.Utilities.Logger;
+using Module = TaleWorlds.MountAndBlade.Module;
+
+namespace Alliance.Server.Patch.HarmonyPatch
+{
+	class Patch_Module
```

Comment:
It seems you are patching `ShutDownWithDelay`. Why class name is `Patch_Module` then?

Reply:
I've always put the native class name in the patch name. Then the patch is named after the native method + type of patch (Prefix/Postfix).
So in this case, it is the Patch_Module class, patching the Module class ^^
And the method is called Postfix_ShutDownWithDelay, since its a post fix for the Module.ShutDownWithDelay method